wg-cloud-provider/draft-20171128-controller-manager.md

```diff
@@ -0,0 +1,294 @@
+---
```

Comment:
you might still want to enclose the YAML in back ticks so it renders a bit nicer